Silicon/MeteorlakePkg/Include/Library/BootGuardLib.h

```diff
@@ -0,0 +1,126 @@
+/** @file
+
+  Copyright (c) 2023, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef _BOOT_GUARD_LIB_H_
+#define _BOOT_GUARD_LIB_H_
```

Comment:
Should use the header file in common folder instead of adding to platform folder?